src/util/localeHelpers.js

```diff
@@ -8,9 +8,9 @@
 
 import { preferredLocales } from '../i18n';
 
-const getLocaleURL = (newLocale, locale, location) => {
+const getLocaleURL = (newLocale, locale, location = {}) => {
   const { pathname, search } = location;
-  const basePath = pathname.startsWith(`/${locale}/`)
+  const basePath = pathname?.startsWith(`/${locale}/`)
```

Comment:
Det gir ikke helt mening å anta at `pathname` kan være `undefined`, for hvis `pathname` er `undefined` så blir `basePath` `undefined` og det gjør at du returnerer en invalid url.

Reply:
Sant. Dette var et forsøk på å gjøre noko som feila lokalt litt meir robust. Men det skulle gjerne feile lokalt.